src/app.zig

```diff
@@ -76,9 +82,18 @@ pub const App = struct {
 
     pub fn update(self: *App, event: Event) !void {
         switch (event) {
+            // TODO: Separate keybind logic
             .key_press => |key| {
                 if (key.matches('c', .{ .ctrl = true }))
                     self.should_quit = true;
+                if (key.matches('l', .{ .ctrl = true }))
+                    if (self.state.activePanel != Panel.calendar) {
+                        self.state.activePanel = Panel.calendar;
+                    };
+                if (key.matches('h', .{ .ctrl = true }))
+                    if (self.state.activePanel != Panel.events) {
+                        self.state.activePanel = Panel.events;
+                    };
```

Comment:
We should probably for ease of use also allow arrows.

Reply:
I'm going to separate the keybinds module so I'll add the arrow keys then. Also this way of doing it won't work, I'll separate the "actions" and then bind each key to an action.